js/interface.js

```diff
@@ -307,32 +311,113 @@
     return Fliplet.API.request({
       url: [
         'v1/widgets?include_instances=true&tags=type:menu',
+        '&include_all_versions=true',
         '&appId=' + Fliplet.Env.get('appId'),
-        '&organizationId=' + (Fliplet.Env.get('organizationId') || '')
+        '&organizationId=' + (organizationId || '')
```

Comment:
There's really no need for this change. The variable `organizationId` is only used once.